base/string_utils.hpp

```diff
@@ -497,27 +497,31 @@ size_t EditDistance(TIter const & b1, TIter const & e1, TIter const & b2, TIter
 {
   size_t const n = distance(b1, e1);
   size_t const m = distance(b2, e2);
+
+  if (m > n)
+    return EditDistance(b2, e2, b1, e1);
+
   // |cur| and |prev| are current and previous rows of the
   // dynamic programming table.
   vector<size_t> prev(m + 1);
-  vector<size_t> cur(m + 1);
+  vector<size_t> curr(m + 1);
```

Comment:
I didn't get your comment. I just fixed the name of the variable, but I'm not planning to fix this code - it's fine. My plan is to implement a function that will check that two strings match in presence of a fixed number of errors.